testing/jormungandr-integration-tests/src/common/jormungandr/process.rs

```diff
@@ -209,10 +285,13 @@ impl SyncNode for JormungandrProcess {
     }
 
     fn get_lines_with_error_and_invalid(&self) -> Vec<String> {
-        self.logger.get_lines_with_error_and_invalid().collect()
+        self.logger
+            .get_lines_with_level(LogLevel::ERROR)
+            .map(|x| x.to_string())
+            .collect()
```

Comment:
This will no longer catch invalid JSON, or does the test runner panic when failing to parse a line as JSON?

Reply:
I see no reason we should have invalid json in the logs, the logger panics if the line cannot be parsed

Reply:
OK